djangocms_versioning/plugin_rendering.py

```diff
@@ -26,20 +26,21 @@ def render_plugin(self, instance, context, placeholder=None, editable=False):
             if not versionable:
                 continue
             if self.toolbar.edit_mode_active or self.toolbar.preview_mode_active:
-                # FIXME Currently this leaves multiple content object
-                # for a single grouper and is potentially dangerous
-                # if content should have some unique constraint
-                # (for example ('page', 'language') for PageContent).
                 qs = versionable.content_model._base_manager.filter(
                     versions__state__in=(DRAFT, PUBLISHED),
                 ).order_by('versions__state')
             else:
                 qs = versionable.content_model.objects.all()
             related_field = getattr(instance, field.name)
             if related_field:
-                qs = qs.filter(**{
-                    versionable.grouper_field_name: related_field
-                })
+                filters = {
+                    versionable.grouper_field_name: related_field,
+                }
+                # TODO Figure out grouping values-awareness
```

Comment:
is there a story for this bit?

Reply:
There's no

Reply:
So the config variable is only valid for `language`? Should there be an exception if you try to pass anything else, or it is not necessary as it will be ignored in these filters?